db/migrate/20241129084054_add_email_templates.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddEmailTemplates < ActiveRecord::Migration[7.1]
+  def change
+    create_table :email_templates do |t|
```

Comment:
@wrmk , @Lyams  why this table doesn't have tenant_id?

Reply:
will add

Reply:
https://github.com/freeats/freeats/pull/284